app/Http/Controllers/ConsumablesController.php

```diff
@@ -273,47 +273,4 @@ public function postCheckout($consumableId)
 
     }
 
-
-    /**
-    * Returns a JSON response containing details on the users associated with this consumable.
```

Comment:
I was already working on the consumables API controller - wish you'd have let me know you were poking at it. Really I need any outstanding importer issues handled.